aggregator/src/aggregator/aggregate_init_tests.rs

```diff
@@ -43,6 +40,8 @@ use std::sync::Arc;
 use trillium::{Handler, KnownHeaderName, Status};
 use trillium_testing::{prelude::put, TestConn};
 
+use super::http_handlers::AggregatorHandlerBuilder;
```

Comment:
My rust-analyzer defaults to this when auto-importing so I haven't thought too critically of it.

What's the rationale for it, so we can put it in our style docs, along with hopefully the rust-analyzer setting to prefer crate::? (I imagine it's so that we can move modules around without having to fiddle with imports?)

Reply:
It looks like `"rust-analyzer.imports.prefix": "crate"` will keep auto-imports from using super.